### PR TITLE
Page for creating a new user

### PR DESCRIPTION
**Pull request by whaught**
## Proposed Changes

* Allows anyone to create a new User for authentication
    * If that user doesn't also exist in our custom users table, it results in a hilarious redirect loop

**Release Note**

```release-note
Create a custom create auth page
```

### CODE REVIEW

File: cmd/server/assets/login/login.html [hunk 1]
```diff
@@ -32,7 +32,8 @@
                 <button type="submit" id="submit" class="btn btn-primary btn-block">Login</button>
               </form>
               <div class="card-body">
-                <a class="card-link" href="legacylogin">New user</a>
+                <a class="card-link" href="login/create">New user</a>
```
[the remark this answers is not in the document]
Comment (whaught) [reply cmd/server/assets/login/login.html:35]: I think we should go with this for now so we can kill the firebaseui one. This one logs-in to session correctly and doesn't ask for real-name.

File: cmd/server/assets/login/create.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+<body class="bg-light">
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <div class="d-flex vh-100">
+      <div class="d-flex w-100 justify-content-center align-self-center">
+        <div class="col-sm-6">
+          <div class="card shadow-sm">
+            <div class="card-header">Create New User</div>
```
Comment (sethvargo) [inline cmd/server/assets/login/create.html:19]: Nit: "Create new user"

File: cmd/server/assets/login/create.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+  <script>
+    $(function () {
+      let $form = $('#loginForm');
+      let $submit = $('#submit');
+      let $email = $('#email');
+      let $password = $('#password');
+      let $retype = $('#retype');
+
+      let auth = firebase.auth()
```
Comment (sethvargo) [inline cmd/server/assets/login/create.html:58]: sc

File: cmd/server/assets/login/create.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+      $form.on('submit', function (event) {
+        event.preventDefault();
+
+        let email = $email.val();
+        let pwd = $password.val();
+
+        if (pwd != $retype.val()) {
+          flash("Password and retyped passwords must match", "danger")
```
Comment (sethvargo) [inline cmd/server/assets/login/create.html:67]: sc here and after return

**Pull request merged**